ci-operator/step-registry/ipi/conf/user-workload-monitoring/ipi-conf-user-workload-monitoring-commands.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+cat >> "${SHARED_DIR}/manifest_umw.yaml" << EOF
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
```

Comment:
Existing step touching this ConfigMap:

```console
$ git --no-pager grep cluster-monitoring-config ci-operator/step-registry/
ci-operator/step-registry/ipi/install/monitoringpvc/ipi-install-monitoringpvc-commands.sh:  name: cluster-monitoring-config
```

You'll probably want both steps to try to be more clever about YAML parsing (via Python?) to manage just the settings that their step is about, instead of having each of them claiming complete ownership of the config and possibly stomping the other.

Reply:
Oh, hmm, this one is critical for upgrades.

I'll use `yq` to add modifications